src/tutorial/furthertopics/queues.rst

```diff
@@ -49,27 +51,34 @@ And paste the following into :cylc:conf:`flow.cylc`:
 
 .. note::
 
-   In graph sections backslash (``\``) is a line continuation character i.e. the
+   In graph sections ``&`` is a line continuation character i.e. the
    following two examples are equivalent:
 
-   .. code-block:: cylc
+   .. code-block:: cylc-graph
 
-      foo => bar & \
+      foo => bar &
              baz
 
-   .. code-block:: cylc
+   .. code-block:: cylc-graph
 
       foo => bar & baz
 
-Open the ``cylc gui`` then run the workflow::
+   ``|`` (or), and ``=>`` act in the same way.
+
+Validate, install and play the workflow, then open the ``cylc gui``::
 
-   cylc gui queues-tutorial &
-   cylc play queues-tutorial
+   cylc validate .
+   cylc install --run-name without-queues
+   cylc play queues-tutorial/without-queues
+   cylc gui
 
 You will see that all the ``steak``, ``pasta``, and ``pizza`` tasks are run
 at once, swiftly followed by all the ``ice_cream``, ``cheesecake``,
 ``sticky_toffee`` tasks as the customers order from the dessert menu.
 
+(If you aren't very quick starting the GUI you may find that the entire
+workflow has already run by the time you navigate to it.)
+
```

Comment:
Why not tell them to play from the UI instead

Reply:
Because the UI may not be installed https://github.com/cylc/cylc-doc/pull/388#discussion_r796746372

Reply:
But you're talking about using it! Instead of telling them to `cylc play queues-tutorial/without-queues`, why not tell them to play the workflow using whichever UI?

Reply:
I've made a change which I think fixes this.